### DISCUSSION TITLE
Is it possible to add both AUTH_DB and AUTH_OAUTH at a time ?

### DISCUSSION THREAD

**Discussion by vyshnav-suresh** [apache/superset#26088] [Q&A / Help] [answered]
we have tried several methods to integate both AUTH_DB and AUTH_OAUTH at a time so that users can use either normal login or oauth login.

**Comment by sfirke** [comment, marked as answer]
No, this is not possible. Superset uses Flask-AppBuilder under the hood for authentication and FAB only supports one auth method at a time. Here is a feature request in that repo: https://github.com/dpgaspar/Flask-AppBuilder/issues/1924
